### ISSUE TITLE
[Question] Is copying across drives possible?

### ISSUE THREAD

**Issue by AndreiPashkin** [OneDrive/onedrive-api-docs#1090]
#### Category
- [x] Question
- [ ] Documentation issue
- [ ] Bug

Hello!

In the [docs][1] there is such an example of a copy-request:
```
POST /me/drive/items/{item-id}/copy
Content-Type: application/json

{
  "parentReference": {
    "driveId": "6F7D00BF-FC4D-4E62-9769-6AEA81F3A21B",
    "id": "DCD0D3AD-8989-4F23-A5A2-2C086050513F"
  },
  "name": "contoso plan (copy).txt"
}
```

Notice - `driveId` is specified. Does it mean that use can copy items across different drives?

[1]: https://github.com/OneDrive/onedrive-api-docs/blob/227cdc2136c6fc880a9a4f148d024850e7bd09aa/docs/rest-api/api/driveitem_copy.md

**Comment by chackman** [comment]
Yes within the same geo

**Comment by AndreiPashkin** [comment]
@chackman, is it the same for moving across drives?

Could you also please give a advise how to check possibility of such copying (or moving if it's possible at all) using the API?

**Comment by chackman** [comment]
Yes, same rules for move.

Are you asking about how you check status on a move or copy operation, whether an item can be moved or copied, or something else?

**Comment by AndreiPashkin** [comment]
@chackman 

> Are you asking about how you check status on a move or copy operation, whether an item can be moved or copied, or something else?

The second - whether it's possible to move item I from drive D1 to drive D2.

**Comment by chackman** [comment]
Move and copy have similar semantics.

**Comment by AndreiPashkin** [comment]
@chackman 
Let me rephrase my question: how do I know if it's possible to move or copy item I from drive D1 to drive D2?

**Comment by chackman** [comment]
The API will fail if the operation [was rejected](https://docs.microsoft.com/en-us/graph/api/driveitem-copy?view=graph-rest-1.0&tabs=http#remarks).  Improving the docs to better enumerate the supported cases is on our backlog.

**Comment by AndreiPashkin** [comment]
@chackman, alright, that makes sense, thank you for the answer.
